core/src/test/scala/org/apache/spark/util/JavaUtils.scala

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
This class needs to be in the same package as what it tests, and be called `JavaUtilsSuite`

Reply:
Yep, done.